include/oneapi/dpl/utility

```diff
@@ -13,8 +13,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef __ONEDPL_utility
-#define __ONEDPL_utility
+#ifndef __ONEDPL_UTILITY
```

Comment:
Sorry, but consistency is still suffering. We have 2 underscores vs 1

Reply:
Yes, You re right. But I simple change this name to upper register.
My proposal to change this in separate PR after merge this PR

Reply:
There are 14 files with this defect in guard names.

Reply:
You are talking about consistency. Do you really want to create a separate PR just to remove (or add) 1 underscore?..

Reply:
Yes, I am ready, Why not?

Reply:
Fixed "2 underscores"